lib/model-access-wrapper.js

```diff
@@ -607,7 +607,22 @@ class ModelAccessWrapper extends CrispHooks {
 	_checkDiffPermissions(permissions, doc, data) {
 		const grant = permissions.getTargetGrant(this.options.permissionsTarget, data);
 		const patch = Update.createFromDiff(doc.data, data);
-		const fields = _.pull(createUpdate(patch).getUpdatedFields(), '_id');
+		const schema = this.model.getSchema();
+		let fields = createUpdate(patch).getUpdatedFields();
+		let defaultedFields = [];
+
+		// Find defaulted fields
+		schema.traverseSchema({
+			onSubschema: (subschema, path) => {
+				if (subschema.default !== undefined && getPath(data, path) === undefined) {
+					defaultedFields.push(path);
+				}
+			}
+		});
+
+		// Skip permissions checks on defaulted fields
+		fields = _.pullAll(fields, defaultedFields);
```

Comment:
That's not true.  It only pulls a field from the fields list if the user hasn't passed anything (`getPath(data, path) === undefined`).  If they pass `CANCELED` it would be in the fields list, and get validated by permissions.